src/__tests__/perseus-markdown-test.jsx

```diff
@@ -63,8 +63,10 @@ var htmlFromMarkdown = function(source) {
 var assertParsesToReact = function(source, html) {
     var actualHtml = htmlFromMarkdown(source);
     if (actualHtml !== html) {
+        /* eslint-disable no-console */
```

Comment:
Oh yeah, I didn't think of that. Yeah, there's a separate `eslintrc.node` in khan-linter, but I don't know what the mechanism would be for specifying that for individual files.